Source/Internal/Table/PDFTableObject.swift

```diff
@@ -284,6 +284,7 @@ internal class PDFTableObject: PDFRenderObject {
         let startPosition: CGPoint = cells.first?.frames.cell.origin ?? .zero
         var nextPageCells: [PDFTableCalculatedCell] = cells
         var pageEnd = CGPoint.null
+        var headerShift = true
```

Comment:
should this be a hardcoded value or should it be connected to `showHeaderOnEveryPage` or `shouldSplitCellsOnPageBreak`?